.github/scripts/test-project.sh

```diff
@@ -67,6 +67,18 @@ fi
 echo "$ cd $dir/$project"
 cd "$dir/$project"
 
+# if FORCE_CUSTOM_OUTPUT is set, we execute commands that will turn the project into a custom output project
+if [ -n "${FORCE_CUSTOM_OUTPUT+x}" ]; then
+  echo "-----------------------------"
+  echo ""
+  echo "FORCE_CUSTOM_OUTPUT=$FORCE_CUSTOM_OUTPUT, executing commands to turn the project into a custom output project"
+  echo ""
+  find . -name "*.js" ! -path "*/node_modules/*" ! -name "*test*" -print0 | xargs -0 sed -i "s/@prisma\/client/db/g"
+  find . -name "*.prisma" ! -path "*/node_modules/*" -print0 | xargs -0 sed -i '/generator client {/a output = "client"'
```

Comment:
I'm not a big fan of coupling the generator name here but at the same time it looks ok at the moment, it's what we use in all tests.

Reply:
You're right, I should not, I could match on the generator provider instead.